Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Black_Hive.xml

```diff
@@ -8,6 +8,36 @@
 		<match Class="PatchOperationSequence">
 			<operations>
 
+	<li Class="PatchOperationConditional">
+	 <xpath>/Defs/ThingDef[defName="AA_BlackSpelopede"]/comps</xpath>
+          		<nomatch Class="PatchOperationAdd">
+            			<xpath>/Defs/ThingDef[defName="AA_BlackSpelopede"]</xpath>
+            			<value>
+              				<comps />
+            			</value>
+		</nomatch>
+	</li>
+
+	<li Class="PatchOperationConditional">
+	 <xpath>/Defs/ThingDef[defName="AA_BlackSpider"]/comps</xpath>
+          		<nomatch Class="PatchOperationAdd">
+            			<xpath>/Defs/ThingDef[defName="AA_BlackSpider"]</xpath>
+            			<value>
+              				<comps />
+            			</value>
+		</nomatch>
+	</li>
+
+		<li Class="PatchOperationAdd">
+			<xpath>/Defs/ThingDef[defName="AA_BlackSpider" or defName="AA_BlackSpelopede"]/comps</xpath>
+			<value>
+				<li>
+				  <compClass>CombatExtended.CompPawnGizmo</compClass>
+				</li>
+				<li Class="CombatExtended.CompProperties_Suppressable" />

```

Comment:
Why would these creatures be suppressable? Base insects aren't.

Reply:
Yes, that two blackhive insects are range combatants, I want add some surpressibility to them to make fire superiority a bit more useful.

Reply:
Hmm. I'll see how it playtests, but I'm not certain of how well it's going to work. I'd suspect they might move to cover but may not be tall enough to effectively fight from cover, depending upon the size of the bugs.

Reply:
They already seek cover by default, making them suppressible will not change  their behaviour.